spread.yaml

```diff
@@ -42,7 +42,7 @@ environment:
     TRUST_TEST_KEYS: '$(HOST: echo "${SPREAD_TRUST_TEST_KEYS:-true}")'
     # a global setting for LXD channel to use in the tests
     LXD_SNAP_CHANNEL: "latest/candidate"
-    UBUNTU_IMAGE_SNAP_CHANNEL: "latest/stable"
+    UBUNTU_IMAGE_SNAP_CHANNEL: "2/stable"
```

Comment:
But this affects all tests, what will happen when we need new features only released in `latest`? For instance, we will need a recompiled u-i when https://github.com/snapcore/snapd/pull/12753 is merged so it support labels in capitals.

Reply:
I believe this was only temporarily, but @sergiocazzolato may correct me here, until the ubuntu-image in latest has been fixed

Reply:
@MiguelPires @alfonsosanchezbeato, this is temporal until latest/stable is fixed, I already raised the issue with @sil2100